src/controllers/user.controller.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
+/* eslint-disable no-useless-return */
```

Comment:
please do not disable this rule - `no-useless-return`
it saying that you doing return when it doesn't make any sense